app/renderer/components/dialogs/FileCreateDialog.jsx

```diff
@@ -65,8 +65,18 @@ export default class FileCreateDialog extends PureComponent<Props> {
   }
 
   onChangeName(e) {
+      const illegalCharacters = /(\\)|(\/)|(\:)|(\;)|(\*)|(\?)|(")|(')|(,)|(\.)|(\<)|(\>)|(\|)|( )/gi;
```

Comment:
Space character should be allowed, as it's not illegal for file/dir names and is used quiet often by users to name files (on Windows at least).

Reply:
https://github.com/oxygenhq/oxygen-ide/commit/3c834427a2c21b38e58171245d206f6d18a18852